toolchains/vita/Dockerfile.m4

```diff
@@ -62,10 +62,17 @@ vdpm_package(freetype)
 
 helpers_package(fribidi)
 
+vdpm_package(FluidLite)
+
 vdpm_package(sdl2)
 
 vdpm_package(sdl2_net)
 
+# The following packages are not used in the current development version.
```

Comment:
Could you reword this comment?
I understand from it that vita2dlib_fbo isn't used anymore but I think it's still the case.

Reply:
`vita2dlib_fbo` was only used by older versions of SDL and the custom backend code that was removed in scummvm/scummvm@044ea0b60e96ccdd6be9187ebcbea3d811a54410.

Reply:
Oh! I didn't see that. It's all good then.
Thanks!